src/main/java/Ending.java

```diff
@@ -0,0 +1,17 @@
+class Ending extends Event {
```

Comment:
Perhaps you may use nouns form for this class name :smiley:
```suggestion
Ending
```